pkg/server/transform.go

```diff
@@ -173,42 +173,42 @@ func messageSliceFromProto(m []*translatev1.Message) ([]model.Message, error) {
 
 // ----------------------Messages----------------------
 
-// messagesToProto converts *model.Messages to *translatev1.Messages.
-func messagesToProto(m *model.Messages) *translatev1.Messages {
-	if m == nil {
+// messagesToProto converts *model.Translation to *translatev1.Translation.
+func messagesToProto(t *model.Translation) *translatev1.Translation {
+	if t == nil {
 		return nil
 	}
 
-	return &translatev1.Messages{
-		Language: languageToProto(m.Language),
-		Original: m.Original,
-		Messages: messageSliceToProto(m.Messages),
+	return &translatev1.Translation{
+		Language: languageToProto(t.Language),
+		Original: t.Original,
+		Messages: messageSliceToProto(t.Messages),
 	}
 }
 
-// messagesFromProto converts *translatev1.Messages to *model.Messages.
-func messagesFromProto(m *translatev1.Messages) (*model.Messages, error) {
-	if m == nil {
+// messagesFromProto converts *translatev1.Translation to *model.Translation.
+func messagesFromProto(t *translatev1.Translation) (*model.Translation, error) {
+	if t == nil {
 		return nil, nil
 	}
 
 	var (
 		err  error
-		msgs = &model.Messages{Original: m.Original}
+		msgs = &model.Translation{Original: t.Original}
```

Comment:
Don't know if you are supposed to change variables in functions as well (as we dont have messages anymore, but have translation, leaving `msgs, messages, e.t.c.`, could be a little confusing later on)

Nevertheless that would be a tedious task to do across all codebase...
```suggestion
		translation = &model.Translation{Original: t.Original}
```

Reply:
The change applies to everything - variables, function names etc. Everything MUST be done in a single PR.